### PR TITLE
seed bundle tests

### PR DESCRIPTION
**Pull request by neonphog**
DEPENDS ON https://github.com/holochain/lair/pull/48

- adds unit and fixture implementation tests

### CODE REVIEW

File: crates/hc_seed_bundle/tests/execute_fixture_tests.rs [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+                    );
+                    let seed = cipher.unlock((a1, a2, a3)).await.unwrap();
+                    let pub_key = seed.get_sign_pub_key();
+                    assert_eq_b64(&self.sign_pub_key, &*pub_key.read_lock());
+                    out = Some(seed);
+                }
+                LockedSeedCipher::UnsupportedCipher(name) => {
+                    if &*name == "securityQuestions" {
+                        println!("HAVEN'T IMPLEMENTED securityQuestions YET");
+                    } else {
```
Comment (maackle) [inline crates/hc_seed_bundle/tests/execute_fixture_tests.rs:123]: Haven't you?

**Review by maackle — APPROVED** [review]

**Comment by maackle** [comment]
Just to make sure, do these tests do a roundtrip of the custom deserialization you wrote?

File: crates/hc_seed_bundle/tests/execute_fixture_tests.rs (hunk 1 above)
Comment (neonphog) [reply crates/hc_seed_bundle/tests/execute_fixture_tests.rs:123]: hah, woops. Good catch : )

**Comment by neonphog** [comment]
@maackle 

> Just to make sure, do these tests do a roundtrip of the custom deserialization you wrote?

Yep, `test_pwhash_cipher` does a round trip of a bundle containing a pwhash cipher, and `test_security_questions_cipher` does a round trip of a bundle containing a security questions cipher.

**Pull request merged**